tests/tests_pytorch/accelerators/test_tpu.py

```diff
@@ -302,7 +301,6 @@ def test_warning_if_tpus_not_used(tpu_available):
         Trainer(accelerator="cpu")
 
 
-@RunIf(tpu=True, standalone=True)
```

Comment:
this should probably stay a standalone test so to isolate access to xla device from other tests.

Reply:
It no longer runs on TPUs